src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/SnikeIKøenService.kt

```diff
@@ -151,8 +152,8 @@ class SnikeIKøenService(
 }
 
 enum class SettPåMaskinellVentÅrsak(val beskrivelse: String) {
+    LOVENDRING("Lovendring"),
     SATSENDRING("Satsendring"),
-    MÅNEDLIG_VALUTAJUSTERING("Månedlig valutajustering"),
```

Comment:
Skal ikke Månedlig valutajustering lenger være en årsak til at behandling settes på maskinell vent? 🤔

Reply:
Var usikker på om den var like relevant for KS nå i førsteomgang, bare. Legger den tilbake 👍